src/button/button.scss

```diff
@@ -33,6 +33,7 @@ $iui-button-padding-large: $iui-xs * 6;
   }
 
   > .iui-icon:not(.iui-user-icon) {
+    display: inline-block;
```

Comment:
Not really seeing any influnce with this one..

Reply:
I think that's because it is a flex child.

Removed.